orchestrator.go

```diff
@@ -7,10 +7,12 @@ import (
 	"os"
 	"os/exec"
 	"strings"
-
+    "cloudfix-linter/logger"
 	tfjson "github.com/hashicorp/terraform-json"
 )
 
+var log=logger.NewLogger()
```

Comment:
This log object is not used anywhere to demonstrate that it works. you should have replaced all existing logs with new ones. Also please remove unnecessary lines of code. When you raise a PR review it yourslef first to make sure no lose ends are lying around.

your code is the legacy you leave behind, make it beautiful, not sloppy.

Reply:
Okay, I will replace them.